.travis.yml

```diff
@@ -6,14 +6,18 @@ rvm:
   - 2.3.4
   - 2.2.7
   - jruby
+bundler_args: ''
```

Comment:
this breaks the caching I think since it removes the `--path vendor/bundle` ... do we need it ?